android/src/main/java/com/matejdr/admanager/BannerAdView.java

```diff
@@ -85,13 +85,10 @@ public void requestLayout() {
     }
 
     private boolean isFluid() {
-        AdSize adSize = adView.getAdSize();
-
-        if (adSize == null) {
+        if (this.adSize == null) {
             return false;
         }
-
-        return adSize == AdSize.FLUID || adSize.toString().equals("320x50_mb");
+        return this.adSize.equals(AdSize.FLUID);
```

Comment:
NB: Could be simplified to `AdSize.FLUID.equals(this.adSize);`. This would handle the null check too.